Dockerfile.shell

```diff
@@ -0,0 +1,21 @@
+FROM python:3.5
+
+RUN groupadd user && useradd --create-home --home-dir /home/user -g user user
+WORKDIR /home/user
+
+# To use this container to interact with a different project change
+# "foreman" to the project name you'd like to interact with.
+COPY foreman/requirements.txt .
+
+RUN pip install -r requirements.txt
+
+COPY data_models/dist/data-refinery-models-* data_models/
+
+# Get the latest version from the dist directory.
+RUN pip install data_models/$(ls data_models -1 | tail -1)
```

Comment:
Yes, that should be true because the `data_models/dist` directory is populated by the `python setup.py sdist` command and it names the tarballs it creates in this pattern `<django project name>-<version>.tar.gz`. Since the project name will always be the same the sorting will essentially be on the version numbers. This will work for versions 0-9, but once I hit version 10 this sorting will break. Therefore I'll change this to be `ls -1 | sort --version-sort | tail -1` which will sort on the version numbers properly.

>What if the file that you just copied (data-refinery-models-*) is a directory?
It shouldn't be. The only thing in that directory should be tarballs created by `python setup.py sdist`. If someone sticks a directory in there called `data-refinery-models-*` then this will probably break and I will have no sympathy for them because they shouldn't go sticking directories into dist directories :P